packages/vite/src/node/build.ts

```diff
@@ -246,15 +246,15 @@ export function resolveBuildOptions(raw?: BuildOptions): ResolvedBuildOptions {
     // Support browserslist
     // "defaults and supports es6-module and supports es6-module-dynamic-import",
     resolved.target = [
-      'es2019',
+      'es2020',
       'edge88',
       'firefox78',
       'chrome87',
       'safari13.1'
     ]
   } else if (resolved.target === 'esnext' && resolved.minify === 'terser') {
     // esnext + terser: limit to es2019 so it can be minified by terser
-    resolved.target = 'es2019'
+    resolved.target = 'es2020'
```

Comment:
Comment now differs from code. Is this even still valid/needed?

Reply:
ES2020 is supported in 5.2.0. (https://github.com/terser/terser/issues/643#issuecomment-675584221)
ES2021 is supported in 5.4.0. ([numeric separators](https://github.com/terser/terser/commit/ee965e8d9e8c47583126f1ae11d0da825e55aa89), [logical assignment](https://github.com/terser/terser/commit/5907df8941dcfa4b52dfd588331483923795cd1f))
It seems ES2022 is not still supported. (cf. [arbitrary module namespace identifier names](https://github.com/terser/terser/issues/1137), [static initialization block](https://github.com/terser/terser/issues/1093))

https://github.com/vitejs/vite/blob/e0fe2008b248aec4813efba34db9e644b851943a/packages/vite/package.json#L115
So it could be `es2021`?

Reply:
I assume we want the lowest bound and not the uppest one, would we?
ES2020 is the highest version that is fully supported by Node v14.6

Reply:
Yeah setting to `2021` make sense to me too